app/email/email.go

```diff
@@ -27,6 +27,15 @@ import (
 
 const serverInactivityHours = 100
 
+// Returns category if enabled is true (default false)
+// If "" is returned when enabled is false, the category headers aren't attached to the email
+func getSendGridCategory(category string, enabled bool) string {
+	if enabled {
+		return category
+	}
+	return ""
+}
```

Comment:
I think we could move this check a bit lower down the stack to avoid having to call this function every time before `es.sendMailXXX` . 

It looks like eventually, these 4 methods are responsible for calling the final mail package functions:
`sendEmailWithCustomReplyTo`
`sendMailWithCC`
`SendMailWithEmbeddedFilesAndCustomReplyTo`
`SendMailWithEmbeddedFiles`

I am wondering if we can just shift the check to there.

On a side note: the extremely large param list is a big eyesore. But perhaps out of scope for this PR. :)